templates/result.html

```diff
@@ -30,5 +31,11 @@ <h5 id ="re-card-title" class="card-title">{{loc.AddressInfo.Title}}</h5>
     </ul>
   </div>
   {% endfor %}
+  {% else %}
+   <div class=" card mt-5 alert alert-warning text-center">
+        <h2 class= 'text-black-50'> No result Found.</h2>
+        <h4 class= 'text-black-50'>May be Location is not correct or try with State Name</h4>
```

Comment:
I would write something like `Location may not be correct. Try including state in your search, for example "New York NY"` - or something like this. Examples of correct queries are generally useful.

Reply:
Yeah, Got It.
And from your this feedback -For the map images - can you link to an actual address of the charging station? Right now all the maps link to the same URL (https://map.openchargemap.io/#/search), which isn't very useful.
I think I will need to use Google Map API to locate the correct location. Open Charge Map doesn't provide that functionality. So Is it fine If I do it later.